CIP-????/README.md

```diff
@@ -0,0 +1,334 @@
+---
+CIP: ?
+Title: Validation zones
+Category: Ledger
+Status: Proposed
+Authors:
+    - Michael Peyton Jones <michael.peyton-jones@iohk.io>
+Implementors: []
+Discussions:
+Created: 2024-03-11
+License: CC-BY-4.0
+---
+
+## Abstract
+
+We propose a set of changes that revolve around validation zones, a construct for allowing greater interdependency between transactions. 
```

Comment:
"a construct for specifying a new type of dependency structure between transactions"